docs/examples.md

```diff
@@ -31,128 +42,94 @@ new CheckmarxSecurityJobBuilder(
-            description: 'this a custom flow job',
-            jobFlow: """
-                build('job1')
-                build('job2')
-                parallel(
-                 { build('job3') },
-                 { build('job4') },
-                )
-            """
-    ).build(this);
-
-    customFlowJob.with{
-      logRotator{
-          numToKeep(365)
-      }
-
-    }
+FlowJobBuilder(
+        name: 'sample-flow-job',
+        description: 'A sample build flow job that builds a list of jobs in order',
+        jobs:['job1', 'job2']
+).build(this).with {
+  logRotator{
+      numToKeep(365)
+  }
+}
 
+new FlowJobBuilder(
+        name: 'sample-flow-job-with-build-flow',
+        description: 'A sample build flow job that takes a jobFlow string, for full control',
+        jobFlow: """
+            build('job1')
+            build('job2')
+            parallel(
+             { build('job3') },
+             { build('job4') },
+            )
+        """
+).build(this).with {
+  logRotator{
+      numToKeep(365)
+  }
+}
 ```
 
 ## JS Build Job
 
-
 ```
-
 import jenkins.automation.builders.JsJobBuilder
 
-
 String basePath = 'JsJobSamples'
 List developers = ['irina.muchnik@cfpb.gov', 'daniel.davis@cfpb.gov']
 
 def repos = [
         [name: 'jenkins-automation', url: "https://github.com/cfpb/jenkins-automation@2.0"],
-        [name: 'collab', url: "https://github.com/cfpb/jenkins-automation", shallow: true]
+        [name: 'collab', url: "https://github.com/cfpb/collab", shallow: true]
         [name: 'other', url: "https://github.com/cfpb/jenkins-automation", disable_submodule: true]
 ]
 folder(basePath) {
     description 'This example shows how to create jobs using Job builders.'
 }
 
-
 new JsJobBuilder(
         name: "$basePath/BuilderVsBuilders",
-        description: 'An example using a job builder for a Javascript build jobs project.',
+        description: 'An example using a job builder for a Javascript build jobs project',
         repos: repos,
         emails: developers,
         use_versions: true
 ).build(this)
-
 ```
+
 ## Using MultiScm Utility 
 
 ```
-
-import javaposse.jobdsl.dsl.DslFactory
-import javaposse.jobdsl.dsl.Job
 import jenkins.automation.builders.BaseJobBuilder
 import jenkins.automation.utils.ScmUtils
 
- def baseJob = new BaseJobBuilder(
-                name: this.name,
-                description: this.description,
-                emails: this.emails
-        ).build(factory)
-
-
- baseJob.with {
+def repos = [
+        [name: 'jenkins-automation', url: "https://github.com/cfpb/jenkins-automation@2.0"],
+        [name: 'collab', url: "https://github.com/cfpb/collab", shallow: true]
+]
+new BaseJobBuilder(
+        name: "sample-job-with-multiscm",
+        description: "A sample with multiple source control repositories",
+).build(this).with {
     multiscm {
-        ScmUtils.project_repos(delegate, this.repos, use_versions)
-        }
+        ScmUtils.project_repos(delegate, repos, true)
```

Comment:
Our typical pattern is to [pass `repos` into the constructor](https://github.cfpb.gov/CFPB/cfgov-jenkins-automation/blob/master/jobs/cfgov-deploy-apache-conf.groovy#L16), which elimintates the need for the `multiscm` block in the `with`.

Should we do that in this example?